bundle/jsky.app.ot/src/main/resources/jsky/app/ot/gemini/altair/AltairForm.jfd

```diff
@@ -169,7 +169,7 @@
       </void> 
       <void method="add"> 
        <object class="com.jformdesigner.model.FormComponent"> 
-        <string>jsky.util.gui.OptionWidget</string> 
+        <string>javax.swing.JRadioButton</string>
```

Comment:
I think the jfd files can go away entirely. They're a relict of the JFormDesigner times.